debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -1202,6 +1202,8 @@ public void readSchema(Tables tables, String databaseCatalog, String schemaNameP
      * included in column.include.list.
      */
     protected Optional<ColumnEditor> readTableColumn(ResultSet columnMetadata, TableId tableId, ColumnNameFilter columnFilter) throws SQLException {
+        final String defaultValue = columnMetadata.getString(13);
```

Comment:
@ani-sha Could you please just add a comment explaining that this is Oracle specific so it must stay there?

Reply:
See `io.debezium.connector.oracle.OracleConnection.readTableColumn(ResultSet, TableId, ColumnNameFilter)`. Also there should be an incubator rep pr that would remove this method as it should be same as in the core

Reply:
Sure thing. Will add them.